### PR TITLE
[PT-66] Generate moonpay primary nft checkout api code

### PR DESCRIPTION
**Pull request by deepti-imx**
# Summary

# Why the changes

# Things worth calling out

# After submitting 
- [ ] ***For Immutable developers:*** Do the changes in this PR require documentation updates? Please refer to this [SDK documentation guide](https://immutable.atlassian.net/wiki/spaces/PPS/pages/1916994017/SDK+documentation+guide) and list links to documentation update PRs (they don't have to be merged) below (or write `N/A`):
    - [ ] *Add documentation update 1*
    - [ ] *Add documentation update 2*

### CODE REVIEW

**Review by CBDB458 — APPROVED** [review]

File: src/api/.openapi-generator/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-6.2.1-SNAPSHOT
+6.3.0-SNAPSHOT
```
Comment (felipetavares) [inline src/api/.openapi-generator/VERSION:1]: @felipetavares reminder to regenerate the secondary flow to update this after we merge

File: src/workflows/minting.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export async function mintingWorkflow(
     ether_key: user.user,
     tokens: user.tokens.map(token => ({
       id: token.id,
-      ...(token.blueprint && { blueprint: token.blueprint }),
+      blueprint: token.blueprint,
```
Comment (felipetavares) [inline src/workflows/minting.ts:24]: Why this change?

Comment (deepti-imx) [reply src/workflows/minting.ts:24]: generating api code brought change to `blueprint` property and it not optional anymore.
This code required fix corresponding to that change.

**Review by felipetavares — APPROVED** [review]
